tensorboard_plugin_3d/plugin.py

```diff
@@ -27,14 +11,23 @@
 from tensorboard.plugins import base_plugin
 from tensorboard.plugins.scalar import metadata
 
-_SCALAR_PLUGIN_NAME = metadata.PLUGIN_NAME
-_PLUGIN_DIRECTORY_PATH_PART = "/data/plugin/example_raw_scalars/"
+def decorate_headers(func):
+    def wrapper(*args, **kwargs):
+        headers = func(*args, **kwargs)
+        headers.extend(TensorboardPlugin3D.headers)
+        return headers
+    return wrapper
+
+exceptions.HTTPException.get_headers = decorate_headers(exceptions.HTTPException.get_headers)
 
+_PLUGIN_DIRECTORY_PATH_PART = "/data/plugin/tensorboard_plugin_3d/"
 
-class ExampleRawScalarsPlugin(base_plugin.TBPlugin):
-    """Raw summary example plugin for TensorBoard."""
 
-    plugin_name = "example_raw_scalars"
+class TensorboardPlugin3D(base_plugin.TBPlugin):
+    """TensorBoard plugin for 3D rendering."""
+
+    plugin_name = "tensorboard_plugin_3d"
+    headers = [("X-Content-Type-Options", "nosniff")]
```

Comment:
Out of curiosity, why was this needed?

Reply:
This is to avoid a future error from Tensorboard. Without this header the following warning is currently returned:
`In 3.0, this warning will become an error: X-Content-Type-Options is required to be "nosniff"`